Policies/github-terms-of-service.md

```diff
@@ -127,45 +128,58 @@ You agree not to engage in activity that significantly harms our Users. We will
 
-This license does not grant GitHub the right to sell your Content or otherwise distribute it outside of our Service.
+This license does not grant GitHub the right to sell Your Content or otherwise distribute or use it outside of our provision of the Service.
 
 #### 5. License Grant to Other Users
-Any Content you post publicly, including issues, comments, and contributions to other Users' repositories, may be viewed by others. By setting your repositories to be viewed publicly, you agree to allow others to view and "fork" your repositories (this means that others may make their own copies of your Content in repositories they control).
+Any User-Generated Content you post publicly, including issues, comments, and contributions to other Users' repositories, may be viewed by others. By setting your repositories to be viewed publicly, you agree to allow others to view and "fork" your repositories (this means that others may make their own copies of Content from your repositories in repositories they control).
 
-If you set your pages and repositories to be viewed publicly, you grant each User of GitHub a nonexclusive, worldwide license to access your Content through the GitHub Service, and to use, display and perform your Content, and to reproduce your Content solely on GitHub as permitted through GitHub's functionality. You may grant further rights if you [adopt a license](/articles/adding-a-license-to-a-repository/#including-an-open-source-license-in-your-repository).
+If you set your pages and repositories to be viewed publicly, you grant each User of GitHub a nonexclusive, worldwide license to use, display, and perform Your Content through the GitHub Service and to reproduce Your Content solely on GitHub as permitted through GitHub's functionality (for example, through forking). You may grant further rights if you [adopt a license](/articles/adding-a-license-to-a-repository/#including-an-open-source-license-in-your-repository). If you are uploading Content you did not create or own, you are responsible for ensuring that the Content you upload is licensed under terms that grant these permissions to other GitHub Users.
 
 #### 6. Contributions Under Repository License
-Whenever you make a contribution to a repository containing notice of a license, you license your contribution under the same terms, and you agree that you have the right to license your contribution under those terms. If you have a separate agreement to license your contributions under different terms, such as a contributor license agreement, that agreement will supercede.
+Whenever you make a contribution to a repository containing notice of a license, you license your contribution under the same terms, and you agree that you have the right to license your contribution under those terms. If you have a separate agreement to license your contributions under different terms, such as a contributor license agreement, that agreement will supersede.
```

Comment:
@nsqe I'm a little confused. I'm not sure I can reconcile your earlier statements that
> we're certainly not intending to be sneakily clever, or sneaky in any way

with your thumbs up to @kaithar's response. IMHO, sneaky clever, even "in a good way" as @kaithar put it, is inappropriate for a legal contact. 

Let me restate what's at issue here. The original wording says:

> Whenever you make a contribution ... you license your contribution under the same terms .. If you have a separate agreement ... that agreement will supersede 

This gives the contributor a choice: follow github's in=out scheme, or if you really want to, use a different agreement.
This can potentially put both Github's terms of service and contributors at odds with project owners. Without this choice bestowed on contributors, if a contributor tries to contribute something without following the expected process, the project owner can point them to the right procedure and ask them to follow it. It's clear to everyone involved that is no other choice. However, with the choice, the contributor can point the project owner to this document and refuse to follow the proper process. This is not a far fetched scenario. Here are two cases:
[Twitter](https://twitter.com/pierrejoye/status/366067690714038272) and [a removed github comment](https://news.ycombinator.com/item?id=9328183).

By leaving the wording as is, you're throwing Github's full weight behind in=out. Large organizations like Apache and Google that don't use in=out may not view that favorably.

Please consider changing the wording to make your intent clear (even if it is to push for in=out in the short term and force it in the long run).